### PR TITLE
Fix Visual Padding For Course List Elements:

### LINKED ISSUE #209: Bug — Schedule: Visual Padding in Course Elements
**Describe the bug**
Visual padding issue in schedule view with course elements.
Course listing elements in the search are taking too much vertical height and the padding appears inconsistent.

**To Reproduce**
1. load semester
2. click on a course in schedule view

**Expected behavior**
Consistent padding around text content

**Screenshots**
Screen Shot 2020-09-25 at 3 03 45 PM

### PR DESCRIPTION
**Pull request by venkatrsrinivas**
Hello Everyone!
This Is My Pull Request Concerning The Updated Visual Padding For The Course Lists. 
Below You Will Find All Important Details: 

**Formal Issue**

This Pull Request Should Ideally Closes #209.   

**Full Testing Procedure**

*Sample Example(s) To Test Functionality:*
> 1. Run Development Environment w/ Loaded Fall 2020 Semester Data.
> 2. View Course List. 
> 3. Should Be Visual Presented w/ Well-Formatted Course List Padding When Selected/Clicked. 

**All Relevant Photos**

Original:

Screen Shot 2020-10-06 at 16 49 34

Final: 

Screen Shot 2020-10-06 at 16 48 01

**Additional Information:**

Thank You!

### CODE REVIEW

File: src/web/src/components/CourseList.vue [hunk 1]
```diff
@@ -32,46 +32,60 @@
...
-              <button
-                v-show="
-                  course.corequisites ||
-                  course.prerequisites ||
-                  course.raw_precoreqs
-                "
-                class="btn"
-                @click.stop="courseInfoModalToggle(course)"
-                data-cy="course-info-button"
+      <template>
```
Comment (123joshuawu) [inline src/web/src/components/CourseList.vue:41]: Don't think you need a template here?

**Review by 123joshuawu — COMMENTED** [review]
Code looks good!

I suggest adding a little padding to the top, left, and bottom of each `CourseListing` element

**Comment by jshom** [comment]
Looks a bit compressed imo, maybe we change to 10px all round?

Heres a small gif with range from current -> 5 all round -> 10 all round
padding-change-yacs

What do you think, do we want a little bit more compact in general?

File: src/web/src/components/CourseList.vue (hunk 1 above)
Comment (venkatrsrinivas) [reply src/web/src/components/CourseList.vue:41]: I'm not sure, I tried to follow the Existing Code that was in place with Recycle Scroller. I'll try to see if removing this Template and its Corresponding one, changes anything.

**Comment by venkatrsrinivas** [comment]
> Great stuff man!
> 
> Small tweak maybe, looks a bit compressed imo, should we change to 10px all round?
> 
> Heres a small gif with range from current -> 5 all round -> 10 all round
> padding-change-yacs
> 
> What do you think, do we want a little bit more compact in general?

**Comment by venkatrsrinivas** [comment]
I think I closed the Pull Request by accident, if that is the case, my bad ...
I think I agree with the Padding, I can add that in. 
Keeping it compact gave us more information but I'm not sure how readable it is.

**Comment by venkatrsrinivas** [comment]
Hello Everyone.
I added 7 px Padding to the Top + Bottom. 
The Left Hand Side needs to be 10 px to match w/ the Right Hand Side Buttons for some reason, or else it just does not look even.

Below is a Screenshot w/ these Minor Changes: 

Screen Shot 2020-10-08 at 19 59 03

Kindly let me know your thoughts.
Thank you.

**Comment by venkatrsrinivas** [comment]
Hello Everyone.
After talking with Jacob, I just made it uniformly 10px. 
Here's what it looks like: 
Screen Shot 2020-10-08 at 20 47 35

Let me know what you think. 
Thank you.

File: src/web/src/components/CourseList.vue (hunk 1 above)
Comment (venkatrsrinivas) [reply src/web/src/components/CourseList.vue]: Hey Josh, 
I just tried it out, and was able to remove it. 
Let me know what you think and if all functionality is as expected. 
Thank you.

**Review by jshom — APPROVED** [review]

**Comment by jshom** [comment]
Looks great!

**Pull request merged**